app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -0,0 +1,283 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material.Card
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import mozilla.components.feature.pocket.recommendations.data.PocketRecommendedStory
+import org.mozilla.fenix.R
+import kotlin.random.Random
+
+/**
+ * Displays a single [PocketRecommendedStory].
+ */
+@Composable
+fun pocketStory(
+    @PreviewParameter(PocketStoryProvider::class) story: PocketRecommendedStory,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier
+            .size(160.dp, 191.dp)
+            .clip(RoundedCornerShape(4.dp))
+            .clickable { /* no-op */ }
+    ) {
+        Card(
+            elevation = 6.dp,
+            shape = RoundedCornerShape(4.dp),
+            modifier = Modifier
+                .size(160.dp, 87.dp)
+                .padding(bottom = 8.dp)
+        ) {
+            // Don't yet have a easy way to load URLs in Images.
+            // Default to a solid color to make it easy to appreciate dimensions
+            Box(Modifier.background(Color.Blue))
+            // Image(
```

Comment:
Don't think the `ThumbnailLoader` can be easily used now to load images in a Composable.
And using `android.widget.ImageView` here doesn't seem like a good approach.

Reply:
Sebastian added a new Jetpack Compose Image Loader in https://github.com/mozilla-mobile/android-components/pull/10838